functions/utils/config.js

```diff
@@ -1,10 +1,13 @@
 const functions = require("firebase-functions");
 
 const auth0Config = functions.config().auth;
+const api = functions.config().api;
 
 const corsConfig = auth0Config ? auth0Config.cors : "";
 const audience = auth0Config ? auth0Config.audience : "";
 const issuer = auth0Config ? auth0Config.issuer : "";
 const jwk_uri = auth0Config ? auth0Config.jwk_uri + ".well-known/jwks.json" : ".well-known/jwks.json";
 
-module.exports = { corsConfig, audience, issuer, jwk_uri };
+const api_token = api ? api.api_token : "";
+const api_url = api ? api.url : "";
```

Comment:
We can remove both of these, since we will be retrieving our M2M token from a DB and the api_url should be the same as the issuer.